### DISCUSSION TITLE
gemini-pro-vision和gpt-4-vision-preview似乎都不支持连续对话？

### DISCUSSION THREAD

**Discussion by kokojacket** [lobehub/lobehub#1429] [General Question] [answered]
### 💻 Operating System

Windows

### 📦 Environment

Docker

### 🌐 Browser

Edge

### 🐛 Bug Description

![image](https://github.com/lobehub/lobe-chat/assets/74887370/4bbdebe0-92ae-4a7d-9efb-2da22de009f5)

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
gpt-4-vision 支持，但 gemini-pro-vision 不支持，跟模型本身能力有关

[a comment by an automated account is not shown]

**Comment by CXwudi** [comment]
gemini-pro-vision是不支持不带图片的请求的
